webpackShims/moment-timezone.js

```diff
@@ -0,0 +1,2 @@
+var moment = module.exports = require('../node_modules/moment-timezone/moment-timezone');
+moment.tz.load(require('../node_modules/moment-timezone/data/packed/latest.json'));
```

Comment:
Definitely outside the scope of this. We currently do `less` and `html` this way and I personally think its fine, it makes it very obvious whats happening.